packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/ccn/1486/indemnite.js

```diff
@@ -0,0 +1,163 @@
+import { round, sum } from "../../math";
+
+const COEFF_ETAM_0 = 1 / 4;
+const COEFF_ETAM_1 = 0.3;
+const MAX_MONTH_INDEMNITE_ETAM = 10;
+const COEFF_IC = 1 / 3;
+const MAX_MONTH_INDEMNITE_IC = 12;
+const COEFF_CE = 1 / 5;
+const MAX_MONTH_INDEMNITE_CE = 7;
+
+function getAncienneteConventionnelle({
+  brancheCategorie,
+  ancienneteLegale,
+  hasBrancheContrat,
+  brancheContrats,
+  brancheAncienneteCE,
+  brancheAncienneteEnqueteur
+}) {
+  let ancienneteConventionnelle = 0;
+  if (brancheCategorie !== "CE") {
+    // categorie is ETAM or IC
+    ancienneteConventionnelle += parseFloat(ancienneteLegale);
+    if (hasBrancheContrat) {
+      ancienneteConventionnelle +=
+        brancheContrats.reduce(
+          (sumContrats, contrat) => sumContrats + parseInt(contrat.duration),
+          0
+        ) / 12; // contrat.duration is in months
+    }
+  } else {
+    ancienneteConventionnelle +=
+      parseInt(brancheAncienneteCE) / 12 + parseInt(brancheAncienneteEnqueteur);
```

Comment:
why parseInt and sometime parseFloat ?

Reply:
Honestly I don't remember, but I guess I should check all these again